lib/apis/gravity.js

```diff
@@ -6,6 +6,6 @@ const { GRAVITY_API_BASE } = process.env;
 
 export default (path, accessToken) => {
   const headers = { 'X-XAPP-TOKEN': config.GRAVITY_XAPP_TOKEN };
-  if (accessToken) assign(headers, { 'X-AUTHENTICATION-TOKEN': accessToken });
+  if (accessToken) assign(headers, { 'X-ACCESS-TOKEN': accessToken });
```

Comment:
Actually, no; and it worked too. So I'm not sure what the difference is, I suppose?